projects/pipelines/testing/config.yaml

```
@@ -190,6 +190,7 @@ base_image:
     bucket_name: pipelines-scale-test-bucket
 tests:
   pipelines:
+    max_concurrent_reconciles: 10
```

```suggestion
    max_concurrent_reconciles: null
```
can you make that null, and then:
```
max_concurrent_reconciles = config.ci_artifacts.get_config("tests.pipelines.max_concurrent_reconciles")
if max_concurrent_reconciles is not None:
  oc patch ...
```
this way, by default, you don't touch RHOAI default value and you keep what *they* prefer

---

Makes sense! Just added that